plugins/hydra_submitit_launcher/example/config.yaml

```diff
@@ -1,4 +1,11 @@
 defaults:
   - hydra/launcher: submitit
+hydra:
+  launcher:
+    params:
+      queue: auto
+      queue_parameters:
+        auto:
+          cluster: local
```

Comment:
Please remove this from the example.
You can add another example showing how to to use the auto queue if you want.

The default queue should be slurm like in the [original plugin](https://github.com/fairinternal/hydra-fair-plugins/blob/master/plugins/hydra-submitit/hydra_plugins/submitit/conf/hydra/launcher/submitit.yaml#L6) (not local as is the case now).
please change that too (it's in config.py).

The test is currently overriding the queue to local so it should continue to work.